### ISSUE TITLE
good staff~~thanks 

### ISSUE THREAD

**Issue by halokid** [acottais/portctl#2]
I can not find the sample of the file "portctl.yaml",  can you give me a sample? and i will try , thank you very much 👍

**Comment by acottais** [comment]
The portctl.yml file used as config file is created when you do a "portctl login" command. It will store the portainer api server URL and also token/credentials to be cached for successive calls. The intent is also to inject this file when using portctl in a container for CI/CD pipelines

Assuming you have a portainer instance running at http://localhost:9000 the command:
`portctl login http://localhost:9000/api`
will create and save the file in the $HOME directory under the name .portctl.yml.

**Comment by halokid** [comment]
yeah, thanks a lot ,  i got it.  i am doing a cli tool base on portianer API to manage docker,  use with jenkins, and i will  make  service functions layout above containers,  your project give me reference, thx :-)
